x-pack/plugins/security_solution/public/entity_analytics/components/severity/common/index.test.tsx

```diff
@@ -13,10 +13,9 @@ import { TestProviders } from '../../../../common/mock';
 import type { EuiHealthProps } from '@elastic/eui';
 import { EuiHealth } from '@elastic/eui';
 
-import { euiThemeVars } from '@kbn/ui-theme';
 import { RiskSeverity } from '../../../../../common/search_strategy';
 import { RiskScoreLevel } from '.';
-import { SEVERITY_COLOR } from '../../../../overview/components/detection_response/utils';
+import { RISK_SEVERITY_COLOUR } from '../../../common/utils';
```

Comment:
RISK_SEVERITY_COLOUR maps to SEVERITY_COLOR for most shades, but since RISK_SEVERITY_COLOUR is more widely used in secsol folders, I chose it for consistency.